### PR TITLE
Obsolete SCL rubygem-faraday_middleware

### PR DESCRIPTION
**Pull request by ehelms**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [ ] 1.19
* [ ] 1.18
* [ ] 1.17

See Foreman's [plugin maintainer documentation](https://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

File: packages/foreman/tfm/tfm.spec [hunk 1]
```diff
@@ -71,6 +71,7 @@ Obsoletes: %{scl_prefix}rubygem-concurrent-ruby < 1.0.1-2
 Obsoletes: %{scl_prefix}rubygem-dalli < 2.6.4-4
 Obsoletes: %{scl_prefix}rubygem-ejs < 1.1.1-4
 Obsoletes: %{scl_prefix}rubygem-flot-rails < 0.0.3-8
+Obsoletes: %{scl_prefix}rubygem-faraday_middleware
```
Comment (ekohl) [inline packages/foreman/tfm/tfm.spec]: All others are versions so we can later re-introduce them with a newer version/release if needed.

Comment (ehelms) [reply packages/foreman/tfm/tfm.spec]: The majority are (https://github.com/theforeman/foreman-packaging/pull/2994/files#diff-707343da1976a0fa7839fb1086a261b9R107). What happens if you need to re-introduce the same version that you are obsoleting?

Comment (ekohl) [reply packages/foreman/tfm/tfm.spec]: They include a release so you can bump the release. See rubygem-flot-rails above. It's `< 0.0.3-8` so you can introduce `0.0.3-8` to replace the old `0.0.3-7` package we used to have.

Comment (ehelms) [reply packages/foreman/tfm/tfm.spec]: I'll follow the majority convention.

**Pull request merged**